frontend/src/bundles/common/components/privat-route/privat-route.tsx

```diff
@@ -0,0 +1,18 @@
+import { Navigate } from 'react-router-dom';
+
+import { AppRoute } from '../../enums/enums';
+import { useAppSelector } from '../../hooks/hooks';
+
+const PrivatRoute: React.FC<{ component: React.FC }> = ({
+    component: Component,
+}) => {
+    const { isLoggedIn } = useAppSelector((state) => state.auth);
```

Comment:
It is better to check if we have fetched user

Reply:
> It is better to check if we have fetched user

How can i do this? Login functionality not yet available

Reply:
- Fixed some logic
- Added a user to the authorization state
- Added user check
- Added a dashboard page and implemented routes for testing
- When registering, authorization occurs